### PR TITLE
Change Cover Guidelines to Image Guidelines

### LINKED ISSUE #2785: "Cover guidelines" link shows up when adding authors images
This should not appear when doing so. Instead, maybe 'author guidelines' could go there.

### PR DESCRIPTION
**Pull request by Sabreen-Parveen**
Closes #2785

### Screenshot
image
cover

### Stakeholders
@mekarpeles @seabelis

### CODE REVIEW

File: openlibrary/templates/covers/add.html [hunk 1]
```diff
@@ -6,15 +6,19 @@
 $if doc.type.key == "/type/author":
     $ intro = _("There are two ways to place an author's image on Open Library")
     $ action = doc.url('/add-photo')
+    $ guideline = _("Image Guidelines")
 $elif doc.type.key == "/type/edition":
     $ intro = _("There are two ways to get a cover image on Open Library")
     $ action = doc.url('/add-cover')
+    $ guideline = _("Cover Guidelines")
 $elif doc.get_edition_covers:
     $ intro = _("There are three ways to get a cover image on Open Library")
```
Comment (cclauss) [inline openlibrary/templates/covers/add.html:15]: Question for reviewers...  Should line 15 say `three ways` or `two ways` like the other instances?

File: openlibrary/templates/covers/add.html [hunk 2]
```diff
@@ -6,15 +6,19 @@
...
     $ action = doc.url('/add-cover')
+    $ guideline = _("Cover Guidelines")
```
Comment (cclauss) [inline openlibrary/templates/covers/add.html:17]:
```suggestion
```
The variables are set the same way in all cases except `$if doc.type.key == "/type/author":`.  This means that we can have one simple if statement — if the doc is an author then we can set the variables one way else we can set the variables another way.

**Review by cclauss — CHANGES_REQUESTED** [review]
We can simplify...

**Comment by Sabreen-Parveen** [comment]
@cclauss I have made the required changes

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Thanks much!

**Pull request merged**